qltool

```diff
@@ -234,7 +234,7 @@ if __name__ == '__main__':
         # attach Qdb at entry point
         if options.qdb == True:
             from qiling.debugger.qdb import Qdb
-            ql.hook_address(Qdb.attach, ql.loader.entry_point)
+            ql.hook_address(Qdb.attach, ql.loader.entry_point & 0xfffffffe)
```

Comment:
for arm / thumb only or all arch ?

Reply:
qdb is now working with MIPS and ARM, since MIPS is aligned by 4 byte its okay to both arch.

Reply:
make it properly and be ready to x86 too :)